codecs/jxl/Makefile

```diff
@@ -78,9 +78,14 @@ $(CODEC_MT_SIMD_BUILD_DIR)/Makefile: CXXFLAGS+=-msimd128
 	-B $(@D) \
 	$(<D)
 
-$(CODEC_DIR)/CMakeLists.txt:
+$(CODEC_DIR)/CMakeLists.txt: 
+	rm -rf $(@D)
 	mkdir -p $(@D)
-	git clone $(CODEC_URL) --recursive -j`nproc` --depth 1 --branch $(CODEC_VERSION) $(@D)
+	cd $(@D) && git init .
+	cd $(@D) && git remote add origin $(CODEC_URL)
+	cd $(@D) && git fetch origin $(CODEC_VERSION) --depth 1
+	cd $(@D) && git checkout $(CODEC_VERSION)
```

Comment:
I'd checkout `FETCH_HEAD` here since the "$(CODEC_VERSION)" may resolve to something different here. For example, if CODEC_VERSION is "master" this will resolve first to the local "master" branch if one exists rather than origin/master.